src/cmake/config.cmake

```diff
@@ -4,11 +4,40 @@ set(PROJECT_NAME "Wazuh Agent")
 # Project version
 set(VERSION "0.1")
 
-# Reading buffer size
-set(BUFFER_SIZE 4096)
+set(DEFAULT_SERVER_URL "https://localhost:27000" CACHE STRING "Default Agent Server Url")
 
-# Default Logcollector file reading interval (ms)
-set(DEFAULT_FILE_WAIT 500)
+set(DEFAULT_REGISTRATION_URL "https://localhost:55000" CACHE STRING "Default Agent Registration Url")
 
-# Default Logcollector reload interval (sec)
-set(DEFAULT_RELOAD_INTERVAL 60)
+set(DEFAULT_RETRY_INTERVAL 30 CACHE STRING "Default Agent retry interval")
+
+set(BUFFER_SIZE 4096 CACHE STRING "Default Logcollector reading buffer size")
+
+set(DEFAULT_FILE_WAIT 500 CACHE STRING "Default Logcollector file reading interval (ms)")
+
+set(DEFAULT_RELOAD_INTERVAL 60 CACHE STRING "Default Logcollector reload interval (sec)")
+
+set(DEFAULT_LOGCOLLECTOR_ENABLED true CACHE BOOL "Default Logcollector enabled")
+
+set(DEFAULT_LOCALFILES "/var/log/auth.log" CACHE PATH "Default localfiles")
```

Comment:
Maybe this doesn't make any sense to be configured this way. We should directly set the default value in `config.h` file.

Reply:
Done